ciao-controller/controller_test.go

```diff
@@ -695,7 +695,12 @@ func TestDetachVolumeFailure(t *testing.T) {
 }
 
 func TestDetachVolumeByAttachment(t *testing.T) {
-	err := context.DetachVolume("invalidTenant", "invalidVolume", "attachmentID")
+	tenant, err := addTestTenant()
```

Comment:
This test is already expected to fail. Which part of the test are you wanting to see expanded?